esx_service/cli/vmdkops_admin.py

```diff
@@ -118,7 +118,8 @@ def commands():
                 '-c': {
                     'help': 'Display selected columns',
                     'choices': ['volume', 'datastore', 'created-by', 'created',
-                                'attached-to', 'policy', 'capacity', 'used'],
+                                'attached-to', 'policy', 'capacity', 'used',
+                                'access', 'attach-as'],
```

Comment:
May be rename the columns as "access-mode" and "attach-mode". Plus documentation on what these choices are    in "vmdkops-admin-cli-spec.md" mentioning what these are,